Demo/Common/IntegrationExamples/DropIn.swift

```diff
@@ -0,0 +1,121 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import AdyenActions
+import AdyenCard
+import AdyenComponents
+import AdyenDropIn
+import AdyenNetworking
+import AdyenSession
+import UIKit
+
+internal final class DropIn: DropInExampleProtocol {
+
+    // MARK: - Properties
+
+    internal var dropInComponent: PresentableComponent?
+    internal weak var presenter: PresenterExampleProtocol?
+    internal var paymentMethods: PaymentMethods?
+    internal var session: AdyenSession?
+    private lazy var containerView = UIView()
+    private lazy var loadingView = LoadingView(contentView: containerView)
```

Comment:
`containerView` and `loadingView ` could be moved to `PresenterExampleProtocol`.

```swift
presenter.showsActivityIndicator = true
```